src/compiler/checker.ts

```diff
@@ -7192,11 +7192,33 @@ namespace ts {
                 markAliasSymbolAsReferenced(symbol);
             }
 
+            const localSymbol = getExportSymbolOfValueSymbolIfExported(symbol);

```

Comment:
`getExportSymbolOfValueSymbolIfExported` returns either a value that was initially passed or a merged symbol for an associated exported symbol so to me `localSymbol` is a bit misleading. In other places we call it similar values `localOrExportedSymbol`